gap/base/methsel.gd

```diff
@@ -19,7 +19,7 @@
 # Our own method selection code:
 
 DeclareInfoClass( "InfoMethSel" );
-SetInfoLevel(InfoMethSel,1);
+SetInfoLevel(InfoMethSel, 0);
```

Comment:
The default level is 0 anyway, so you could just drop this line:
```suggestion
```
However, I note that there is only one situation when we show `InfoMethSel ` messages on level 1, namely when `CallMethods` went over all methods and nothing worked. That "normally" shouldn't happen (unless we encounter a case that we didn't implement yet), so I kinda see why they set this to level 1 by default -- recog is still very incomplete and under development and seeing this output is a valuable clue for what's wrong.

So I feel uneasy about this change. Perhaps we should discuss this and rest on the phone tomorrow.

Reply:
Makes sense. I'm dropping that commit too.